src/_includes/meta.hbs

```diff
@@ -8,11 +8,11 @@
 <meta property="og:site_name" content="Urlaubsverwaltung.cloud" />
 <meta
 	property="og:description"
-	content="Starte heute mit der Digitalisierung deiner HR. Der Urlaubsplaner in der Cloud hilft dir deinen Urlaub zu verwalten und zu beantragen. Ganz einfach. Ohne Papierkram. Ohne Excellisten. Jetzt 30 Tage kostenlos und unverbindlich testen."
+	content="Urlaubsverwaltung dein Urlaubsplaner für Abwesenheiten. Starte heute mit der Digitalisierung deiner HR. Der Urlaubsplaner in der Cloud hilft dir deinen Urlaub zu verwalten und zu beantragen. Ganz einfach. Ohne Papierkram. Ohne Excellisten. Jetzt 30 Tage kostenlos und unverbindlich testen."
```

Comment:
Hm da fehlt glaube ich zumindest ein Komma. Aber das Wort Urlaubsverwaltung muss unbedingt rein. Ich glaube wir sind im Ranking etwas runter gerutscht, weil das nicht mehr in der Description ist. Außerdem werden die Suchwörter in der Description bold dargestellt.